Makefile

```diff
@@ -1,28 +1,28 @@
 .PHONY: examples tests publish
 
 check:
-	@which pylint >/dev/null || pip install --user --upgrade pylint
-	@if python --version | grep -qi 'python 3'; then pylint --reports=no --score=no --disable=R0401 minio/*py; fi
-	@if python --version | grep -qi 'python 3'; then pylint --reports=no --score=no minio/credentials minio/select tests/functional; fi
+	@pip install --user --upgrade pylint==2.5.3
```

Comment:
1. I don't think it is a good idea to stick to a version instead we need to install the latest for github ci.
2. Its not required to install `isort` specifically because pip install pylint automatically resolves it dependencies and we are good with the dependent isort.